fair/models.py

```diff
@@ -1,8 +1,35 @@
 from django.db import models
 from datetime import date
+from lib.image import UploadToDirUUID
+
 
 class Fair(models.Model):
-    name = models.CharField(max_length=100, default="Armada %d"%(date.today().year))
-    year = models.IntegerField(default=date.today().year) 
+    name = models.CharField(
+            max_length=100,
+            default="Armada %d" % (date.today().year)
+            )
+    year = models.IntegerField(default=date.today().year)
+
+    def __str__(self):
+        return self.name
+
+
+class Partner(models.Model):
+    name = models.CharField(max_length=50)
```

Comment:
Hm I guess we could do that, but I though most of the partners are not actually going to the fair (e.g. Asana, Zoegas) so having them as a company would not make any sense?